src/functions-and-arrays.js

```diff
@@ -24,11 +80,22 @@ function sum() {}
 
 // Iteration #4: Calculate the average
 // Level 1: Array of numbers
-const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
-function averageNumbers() {}
+const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
+function averageNumbers(arr) {
+  let media = 0;
+  let soma = 0;
 
+  for (let i =0; i < arr.length; i+=1){
+    soma += arr[i]; 
+    media = (soma / arr.length);
+    
+  }
```

Comment:
Aqui, se vc fizer a média dentro do for, ele vai repetir essa divisão várias vezes! teria de ser depois do for.
Mas aqui também, lembre-se que vc já tem uma função que faz a soma, vc pode utilizá-la da seguinte maneira:
```javascript
function averageNumbers(arr) {
  const soma = sumNumbers(arr);
  const media = soma / arr.length;
  return media;
}
```

Faz sentido?